astropy/units/tests/test_equivalencies.py

```diff
@@ -477,6 +491,24 @@ def test_spectraldensity6():
         snu, sb_flam, u.spectral_density(wave)), sb_fnu, rtol=1e-6)
 
 
+@pytest.mark.parametrize(
+    ('from_unit', 'to_unit'),
+    [(u.ph / u.cm ** 2 / u.s, (u.cm * u.cm * u.s) ** -1),
+     (u.ph / u.cm ** 2 / u.s, u.erg / (u.cm * u.cm * u.s * u.keV)),
+     (u.erg / u.cm ** 2 / u.s, (u.cm * u.cm * u.s) ** -1),
+     (u.erg / u.cm ** 2 / u.s, u.erg / (u.cm * u.cm * u.s * u.keV))])
+def test_spectraldensity_not_allowed(from_unit, to_unit):
+    """Now allowed to succeed as
```

Comment:
Now->Not

Reply:
Ops! Good catch. That does change the meaning completely. 😅